src/app/dashboard/components/dashboard/dashboard.component.html

```diff
@@ -22,7 +22,16 @@ <h1>{{ 'dashboard.heading' | translate }}</h1>
       *ngIf="data.hasSearch || data.hasActions"
     >
       <div *ngIf="data.hasSearch">
-        <h2>{{ 'dashboard.search' | translate }}</h2>
+        <h2>{{ 'dashboard.search.title' | translate }}</h2>
+        <div class="search-box">
+          <erz-typeahead
+            [typeaheadService]="studentsRestService"
+            [placeholder]="'dashboard.search.placeholder'"
+            [value]="filter.student"
+            (valueChange)="$event && navigateToDossier($event)"
+          >
+          </erz-typeahead>
+        </div>
```

Comment:
Sieht gut aus!

Ein paar Rückmeldungen:
- Ich hätte für die `search-box` eine eigene Komponente gemacht, um die Dashboard Komponente selber möglichst schlank zu behalten. Zumal hier nach noch etliches mehr rein kommt. Das würde ich dann auch alles in eigene Komponente tun (eine Actions Komponente und eine Timetable Komponente).
- Braucht es den initialen `[value]` und den `filter` auf der Komponente? Das wird ja nie einen initialen Wert haben...